Spokestack/TextToSpeech.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-@objc public enum TTSInputType: Int {
+@objc public enum TTSInputFormat: Int {
```

Comment:
If this were `TTSInputFormat: String`, you could use `input.inputFormat.rawValue` below and do away with the `switch`.

Reply:
objc interop requires `Int` :````````(